app/src/main/java/grinnell/appdev/edu/lyles/fragments/BeerFragment.java

```diff
@@ -1,22 +1,127 @@
 package grinnell.appdev.edu.lyles.fragments;
 
+import android.os.AsyncTask;
 import android.os.Bundle;
-import android.support.annotation.Nullable;
 import android.support.v4.app.Fragment;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
 
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
 import grinnell.appdev.edu.lyles.R;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+
+public class BeerFragment extends Fragment{
 
-/**
- * Created by Mattori on 5/9/16.
- */
-public class BeerFragment extends Fragment {
+    ArrayList<Beers> beersList;
+    private static final String BEER_DATA_URL ="http://www.cs.grinnell.edu/~birnbaum/appdev/lyles/beer.json";
+    BeersAdapter mBeerAdapter;
+    RecyclerView rvBeers;
 
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
-        return inflater.inflate(R.layout.beer_layout, container, false);
+        final String TAG = BeerFragment.this.getTag();
+
+        View view = inflater.inflate(R.layout.beer_layout,container,false);
+        // ...
+        // Lookup the recyclerview in activity layout
+        rvBeers = (RecyclerView) view.findViewById(R.id.rvBeers);
+
+        // Initialize contacts
+        beersList =new ArrayList<Beers>();
+
+        new BeersAsyncTask(this).execute(BEER_DATA_URL);
```

Comment:
The `BeersAsyncTask` takes a String parameter (here it's `BEER_DATA_URL`), but it doesn't use it. Either change the `doInBackground` method so that it uses its parameters or remove them.